ee/localserver/krypto-ec-middleware.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/kolide/launcher/ee/presencedetection"
 	"github.com/kolide/launcher/pkg/log/multislogger"
 	"go.opentelemetry.io/otel/attribute"
+	"go.uber.org/atomic"
```

Comment:
huh. I think we usually use `sync/atomic`

Reply:
Oh, super weird, that is definitely the one I meant to use! Thank you, I will update